### PR TITLE
[REM] libmy and fix some stuff around

### PR DESCRIPTION
**Pull request by lg-epitech**

### CODE REVIEW

File: gertrude
[the remark this answers is not in the document]
Comment (lg-epitech) [reply gertrude:1]: uh my bad I guess, some rm must've went wrong. I guess, you can just ignore that change
unless you need me to actually push it again let me know.

File: gertrude_base/gertrude_prebuilt.c [hunk 1]
```diff
@@ -16,7 +16,7 @@ void base_mkf(void)
 
     err = system("cp /opt/gertrude-cli/gertrude_prebuilt_mkf/Makefile ./");
     if (err != 0) {
-        my_printf("%sYou need to run gertrude as sudo in order to generate \
```
[the remark this answers is not in the document]
Comment (lg-epitech) [reply gertrude_base/gertrude_prebuilt.c:19]: ehh probably not but it wouldn't display the way you would intend it to before

[the remark this answers is not in the document]
Comment (lg-epitech) [reply gertrude_base/gertrude_prebuilt.c:19]: oh
and yes we do

[the remark this answers is not in the document]
Comment (lg-epitech) [reply gertrude_base/gertrude_prebuilt.c:19]: Although that could probably get sent to the error handling and throw a proper error, will do sometime later

File: gertrude
[the remark this answers is not in the document]
Comment (lg-epitech) [reply gertrude:1]: coming right up

File: gertrude_base/gertrude_prebuilt.c (hunk 1 above)
[the remark this answers is not in the document]
Comment (AshDev05) [reply gertrude_base/gertrude_prebuilt.c:19]: Alr if you say so I trust
Good plant

File: gertrude
[the remark this answers is not in the document]
Comment (lg-epitech) [reply gertrude:1]: It's back now

**Review by AshDev05 — APPROVED** [review]
All good, looks better now

**Pull request merged**